### ISSUE TITLE
Add Support For .exe Packages

### ISSUE THREAD

**Issue by BenVillalobos** [dotnet/arcade#6120]
During development of https://github.com/dotnet/msbuild/pull/5626, we ran into issues trying to publish an `.exe` package. Our workarounds consisted of:

<issue WIP>

**Comment by garath** [comment]
@BenVillalobos I'm not sure what you're looking for here. Is there something you need from Arcade?

**Comment by rainersigwald** [comment]
@garath Yes. Arcade makes a bunch of assumptions in its Visual Studio component authoring process that aren't valid in all circumstances--specifically, it assumes that every VS component is a VSIX container, but that's not always true. We had to spackle over a bunch of stuff to get the correct package build. It would be preferable if it was just supported by Arcade.

**Comment by riarenas** [comment]
I'm still trying to understand what this issue is asking for. The link for what the workaround doesn't seem to be working.

It mentions publishing, do you mean publishing to the dotnetcli container for later consumption?

**Comment by rainersigwald** [comment]
No, this is entirely about _building_ non-VSIX packages. Publishing worked fine.

Current workarounds are

https://github.com/dotnet/msbuild/blob/e89501e67e07fe474a70153e5df3766d27db0085/src/Package/Microsoft.Build.UnGAC/Microsoft.Build.UnGAC.csproj#L7-L8

https://github.com/dotnet/msbuild/blob/e89501e67e07fe474a70153e5df3766d27db0085/src/Package/Microsoft.Build.UnGAC/Microsoft.Build.UnGAC.csproj#L13-L17

**Comment by riarenas** [comment]
> No, this is entirely about building non-VSIX packages.

Thanks! That makes sense. I'm showing my bias when I see publishing mentioned 😄
